themes/bootstrap5/templates/RecordTab/holdingsils/standard.phtml

```diff
@@ -30,6 +30,9 @@
       </span>
       <?php if ($availabilityStatus->isAvailable()): ?>
         <?php /* Begin Available Items (Holds) */ ?>
+        <?php if (($this->config()->displayLoanType() ?? false) && !empty($holding['loan_type_name'])): ?>
+          <span class="loan-type"><?=$this->transEsc('Loan Type')?>: <?=$holding['loan_type_name']?>&nbsp;</span>
```

Comment:
Is the non-breaking space really needed here? It looks okay without it to me, and if extra padding is needed, it could be accomplished through CSS since we've assigned a class.

```suggestion
          <span class="loan-type"><?=$this->transEsc('Loan Type')?>: <?=$holding['loan_type_name']?></span>
```

Reply:
Nope, those aren't needed. That's probably something goofy in our custom code. I didn't even notice it. They're gone now.